supabase-ui/src/components/SideOver/SideOver.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { action, actions } from '@storybook/addon-actions'
+
+import Transition from './../../lib/Transition'
+import { Button } from './../../index'
+
+const SideOver = ({
```

Comment:
"SideOver" is a new one for me, should this be "SlideOver"  (verb) or "SidePanel" (noun)?  

personal preference would be to use nouns for component and verbs for interactions (Transition etc) -  more consistent with our Dashboard

Reply:
I think it was a name from tailwind.
we're not going to use this at the minute, as we need nested SideOvers - but kept it in to update later.

Reply:
You sure it wasnt' "Slide overs"? https://tailwindui.com/components/application-ui/overlays/slide-overs